react/components/GalleryItem.js

```diff
@@ -25,12 +26,22 @@ export default class GalleryItem extends Component {
     const normalizedProduct = { ...product }
     const [sku] = normalizedProduct.items || []
 
+    const sum = array => array.reduce((x, y) => x + y)
```

Comment:
`reduce` should take an initial value. And, since `ramda` is being used, you can use its `sum` function